homeassistant/components/hassio/http.py

```diff
@@ -224,4 +224,10 @@ def should_compress(content_type: str) -> bool:
     """Return if we should compress a response."""
     if content_type.startswith("image/"):
         return "svg" in content_type
+    if content_type.startswith("application/"):
```

Comment:
More general question: Should text/plain also be compressed? (ie log files)

Reply:
Yes absolutely

Reply:
... and we don't actually do that here

https://github.com/home-assistant/core/pull/110865

Reply:
Side node: chrome network panel doesn't show file downloads, but firefox does so much easier to check there